.github/workflows/test.yml

```diff
@@ -102,35 +102,35 @@ jobs:
           make test-integration
         if: env.GIT_DIFF
 
-  integration_tests:
```

Comment:
ignoring test due to unsolved issue in nixOS
https://github.com/nix-community/gomod2nix/issues/20
To reproduce error 
`make run-integration-tests`